sources/Interop/Windows/um/dxcapi/IDxcVersionInfo2.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/dxcapi.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    [Guid("fb6904c4-42f0-4b62-9c46-983af7da7c83")]
+    public unsafe partial struct IDxcVersionInfo2
```

Comment:
The `IID_IDxcVersionInfo2` are missing. They don't exist in native but are convenient to avoid the method call from `typeof(T).Guid`